packages/axelar-wasm-std/src/utils.rs

```diff
@@ -19,6 +22,22 @@ impl<T> TryMapExt<T> for Vec<T> {
     }
 }
 
+/// since the `Felt` type doesn't error on overflow, we have to implement that check
+pub fn does_felt_overflow_from_slice(felt_hex_slice: &[u8]) -> bool {
+    if felt_hex_slice.len() > 32 {
+        return true;
+    }
+    let felt_max_hex_str = format!("{:064x}", Felt::MAX);
```

Comment:
can the corresponding U256 not be defined as a const?

Reply:
I defined that as a const into the new package.